src/app/rugby/[team]/page.tsx

```diff
@@ -0,0 +1,43 @@
+import prisma from '@db/prismaSingleton'
+
+type PageProps = {
+  params: {
+    team: string,
+  },
+}
+
+const Page = async (props: PageProps) => {
+  const {
+    params: {
+      team: teamkey,
+    }
+  } = props
+
+  const players = await prisma.player.findMany({
+    where: {
+      teamMembers: {
+        some: {
+          team: {
+            key: teamkey,
```

Comment:
this could cause a bug because we are getting all players for this team key, even if there's another team with the same key under a different sport

i think we should change the model so `key` is completely unique, not just unique for key-sportId combination

Reply:
Ok yep we can do that or we could additionally pass through the sport in the findMany query. Don't we have access to the sport in the params from the URL?

Reply:
Oh thats only if we used the dynamic sport route as well
[sport]
  [team]
  
  in the folder structure we have stuck with rugby. Was there a reason for this or should we be using the two tiered dynamic directory structure? I thought that was pretty cool

Reply:
we don't need to have the dynamic route to know it's rugby since we know if the user got into this page, it's through the `/rugby` route

in terms of having a dynamic `[sport]` route, then we have to make the page flexible enough to accomodate all sports, i think that's hard work and realistically adding a 2nd sport is a long way off